app/services/album.js

```diff
@@ -22,3 +22,34 @@ exports.getPhotosOfAlbum = albumId => {
       throw errors.conectionError(e.message);
     });
 };
+
+exports.getAllAlbums = (offset, limit, filter, orderBy) => {
+  const endpoint = `${url}albums`;
+  return axios
+    .get(endpoint)
+    .then(response => response.data)
+    .catch(e => {
+      throw errors.conectionError(e.message);
+    })
+    .then(albums => {
+      const processedAlbums = albums.slice();
+      if (filter) {
+        return processedAlbums.filter(album => album.title === filter);
+      }
+      if (orderBy) {
+        processedAlbums.sort((a, b) => {
```

Comment:
this function seems a little bloated, how about making a separate function for sorting this?